### PR TITLE
p2p: autonat status

### PR DESCRIPTION
**Pull request by corverroos**
Adds autonat reachability metrics. Also add panel to simnet dash,

category: feature
ticket: none

### CODE REVIEW

**Comment by corverroos** [comment]
Example dashboard: 
image

File: p2p/p2p.go [hunk 1]
```diff
@@ -182,6 +184,34 @@ func multiAddrViaRelay(relayPeer Peer, peerID peer.ID) (ma.Multiaddr, error) {
 	return transportAddr.Encapsulate(relayAddr), nil
 }
 
+func NewEventCollecter(tcpNode host.Host) lifecycle.HookFuncCtx {
+	return func(ctx context.Context) {
+		sub, err := tcpNode.EventBus().Subscribe(new(event.EvtLocalReachabilityChanged))
+		if err != nil {
+			log.Error(ctx, "Subscribe libp2p events", err)
```
Comment (OisinKyne) [inline p2p/p2p.go:191]: Nit,  probably not worth doing if it's an error deep in prom logs and not user facing  code.  
```suggestion
			log.Error(ctx, "There was an error subscribing to libp2p LocalReachabilityChanged events", err)
```

**Review by OisinKyne — APPROVED** [review]
LGTM

File: p2p/p2p.go (hunk 1 above)
Comment (corverroos) [reply p2p/p2p.go:191]: yeah, this is deep inside, should never happen in practise

**Pull request merged**